src/Xrefcheck/Core.hs

```diff
@@ -180,25 +180,25 @@ pattern PathSep <- (isPathSeparator -> True)
 
 -- | Type of reference.
 data LocationType
-  = LocalLoc
-    -- ^ Reference on this file
+  = CurrentFileLoc
+    -- ^ Reference on this file, e.g. [a](#header)
   | RelativeLoc
-    -- ^ Reference to a file relative to given one
+    -- ^ Reference to a file relative to given one, e.g. [b](folder/file#header)
   | AbsoluteLoc
-    -- ^ Reference to a file relative to the root
+    -- ^ Reference to a file relative to the root, e.g. [c](/folder/file#header
   | ExternalLoc
-    -- ^ Reference to a file at outer site
+    -- ^ Reference to a file at outer site, e.g [d](http://www.google.com/doodles)
   | OtherLoc
-    -- ^ Entry not to be processed (e.g. "mailto:e-mail")
+    -- ^ Entry not to be processed, e.g. "mailto:e-mail"
   deriving stock (Eq, Show)
 
 instance Buildable LocationType where
   build = \case
-    LocalLoc -> color Green "local"
-    RelativeLoc -> color Yellow "relative"
-    AbsoluteLoc -> color Blue "absolute"
-    ExternalLoc -> color Red "external"
-    OtherLoc -> ""
+    CurrentFileLoc -> color Green "local"
```

Comment:
Shouldn't you also update printed text?

Reply:
Oh, thanks, I was choosing between two options and then distracted somehow. I think `"current file"` is ok here

Reply:
Do not forget to add `CHANGES` entry. Also looks like your PR description is missing checklist.